syntax = "proto3";

package int2str_service;

message Arguments {
    int32 argument = 1;
}

message Result {
    string value = 1;
}

service Int2Str {
    rpc int2str(Arguments) returns (Result) {}
}
